docs/user-guide/server-side-install-overview.md

```diff
@@ -1,19 +1,124 @@
+
+* **Zowe System Services (ZSS)** 
+Working closely with the Zowe Cross Memory server (ZIS), ZSS serves as one of the primary, authenticated back-ends that communicates with z/OS and provides Zowe with a number of APIs including z/OS Unix files and data sets, control of the plug-ins and services lifecycle, and security management. ZSS is part of the Application Framework.<details><summary>Click here for more information about Zowe System Services.</summary> The Zowe Desktop delegates a number of its services to ZSS which it accesses through the default http port `7557`. ZSS is written in C and uses native calls to z/OS to provide its services.</details>
+
+## Installation of Zowe runtime
+
+Zowe runtime refers to the full, unarchived set of binaries, executable files, scripts, and other elements that are run when Zowe is started.
+Runtime installation includes the following components:  
+* Zowe Application Framework
+* Zowe API Mediation Layer
+* Z Secure Services (ZSS)
+
+
+**Important:** To run all of these server-side components, or to run Zowe Application Framework, installation of the Zowe Cross Memory Server (ZIS) is required. The Zowe cross memory server (ZIS) provides privileged cross-memory services to the Zowe Desktop and runs as an APF-authorized program.<details><summary> Click here for more information about ZIS.</summary>The same cross memory server can be used by multiple Zowe desktops. The cross memory server is needed to be able to log on to the Zowe desktop and operate its apps such as the Code Editor.</details>
+
+
+For more information about the Zowe cross memory server, see [Installing and configuring the Zowe cross memory server (ZWESISTC)](./user-guide/configure-xmem-server).
+
+## Zowe runtime installation methods and roadmap
+
+To get started with installing Zowe runtime, select your installation method from the following options. (Details of each installation method are described later in this article).
+
+* SMP/E build
+* Convenience build
+* Portable software instance (PSWI)
+* Containerization build  
+    **Note:** The containerization build makes it possible for some UNIX components to run in a container rather than under USS. For more information about this build method, see [Installing Zowe Containers](k8s-introduction.md).
+
+Zowe server-side components can be installed either on z/OS only, or both on z/OS and on Docker.
+
+The following diagram presents a roadmap for installation using the three primary installation methods. If you plan to install these components using a container build, see Installing Zowe Containers.
+
+![Install the Zowe z/OS runtime](../images/install/install.png)
```

Comment:
The install diagram isn't part of the PR, does it already exist?

Reply:
yes. This is published here: https://docs.zowe.org/stable/user-guide/install-zos/#stage-2-installing-the-zowe-zos-runtime